.travis.yml

```diff
@@ -7,8 +7,8 @@ cache:
 
 jobs:
   include:
-    - dart: 2.2.0
-      name: "SDK: 2.2.0"
+    - dart: 2.3.0
```

Comment:
Should we update this?

Reply:
Not sure what you're asking. I updated this because the SDK minimum had to be raised (https://github.com/Workiva/fluri/pull/83/files#diff-fb2d52cd953afedcafc9fe7629c6f132L13) due to my usage of a `for` loop inside a collection.

Reply:
I was suggesting using `2.7.2` since that's what we deploy on

Reply:
Ah I see. I think what we have here is the best option for this package since it's OSS - it tests on the lowest advertised compatible SDK version, stable, and dev.